client/containers/Pub/PubDocument/PubHeaderFormatting.tsx

```diff
@@ -51,6 +51,15 @@ const PubHeaderFormatting = (props: Props) => {
 				}}
 			/>
 			<div className="right-content">
+				<FormattingBar
```

Comment:
This does render a few divs with no content, but some aria tags in the editor. Can you wrap this in the suggested edits flag so it doesn't render anything when the flag is off?